### ISSUE TITLE
Info gathering for Additional Test Clusters for Upgrade testing

### ISSUE THREAD

**Issue by msdisme** [nerc-project/operations#172]
From the 68 nodes currently available for adding to the NERC 10 are to be used to create a test cluster and 10 to create an additional RHODS test cluster.  @dystewart this issue is to identify the information you need to be able to create those clusters- can then use that info to create additional issues as needed.
 
1. systems that work ( @aabaris pointed out that not all the 68 are working)

**Comment by dystewart** [comment]
@msdisme This is the runbook I created when building the test cluster: [https://github.com/nerc-project/nerc-runbooks/blob/nerc-ocp-test-rebuild/docs/nerc-ocp-test-rebuild.md](https://github.com/nerc-project/nerc-runbooks/blob/nerc-ocp-test-rebuild/docs/nerc-ocp-test-rebuild.md)

After looking back over that doc I believe I will need the following in order to build the new clusters:

1. A list of host names ensuring we use the same naming conventions as we did for the other clusters (assuming that is the intention going forward). 

See specifically the shell script where we use ipmitool to pxe boot the hosts: [https://github.com/nerc-project/nerc-runbooks/blob/nerc-ocp-test-rebuild/docs/nerc-ocp-test-rebuild.md#bootstrapping-nerc-ocp-test](https://github.com/nerc-project/nerc-runbooks/blob/nerc-ocp-test-rebuild/docs/nerc-ocp-test-rebuild.md#bootstrapping-nerc-ocp-test)
(avoided pasting the scrtipt in this issue since it's from a private repo)

2. Configure DNS for the api ip and ingress ip of the cluster. For example running
```
$ nslookup api.nerc-ocp-<cluster_name>.rc.fas.harvard.edu
$ nslookup lb.nerc-ocp-<cluster_name>.rc.fas.harvard.edu
```
should return ip addresses for the aforementioned api and ingress

3. I believe there will be some external ceph config that needs to happen @jtriley and @larsks probably have more insight on this topic.

I think those 3 points give us a pretty good runway to get started.
